src/main/webapp/app/overview/student-questions/student-question-answer/student-question-answer.component.html

```diff
@@ -0,0 +1,111 @@
+<div class="row mb-1 pl-2 answer" [ngClass]="{ 'answer-approved': studentQuestionAnswer.tutorApproved }">
+    <div class="col-12">
+        <!-- edit Mode on -->
+        <ng-container *ngIf="isEditMode">
+            <div class="row mb-2">
+                <div class="col-12">
+                    <div class="row mb-1">
+                        <div class="col px-0">
+                            <span class="form-control-label">{{ 'artemisApp.courseOverview.exerciseDetails.faq.editAnswer' | translate }}</span>
+                        </div>
+                        <div class="col-auto px-0" *ngIf="isAtLeastTutorInCourse || isAuthorOfAnswer(studentQuestionAnswer)">
+                            <fa-icon
+                                class="mr-1 clickable"
+                                [ngbTooltip]="'artemisApp.courseOverview.exerciseDetails.faq.editAnswer' | translate"
+                                container="body"
+                                [icon]="'pencil-alt'"
+                                (click)="$event.stopPropagation(); isEditMode = false"
```

Comment:
I think angular events do not bubble like normal javascript dom events -> event.stopPropagagtion is not needed